.gitignore

```diff
@@ -6,7 +6,7 @@ docker-build
 **/target/
 /storage/
 /test-utils/node/storage
-/test-utils/sandbox/storage
+/test-utils/ganache/storage
```

Comment:
Why do we change it from sandbox to ganache?

Reply:
ganache is correct one: https://github.com/near/nearcore/blob/master/.gitignore#L9. I accidentally replaced it with sandbox so i double check every ganache->sandbox replace is doing correct.